gateway/1.9/modules/ROOT/pages/flex-architecture-shared-storage.adoc

```diff
@@ -28,5 +28,6 @@ image:redis-same-vpc.png["The redis shared storage is in the same VPC as the Fle
 
 == See Also
 
-* xref:flex-architecture-basic-deployments.adoc[]
-* xref:flex-architecture-uc-overview.adoc[]
\ No newline at end of file
+* xref:flex-conn-shared-storage-config.adoc[Configure Shared Storage in Connected Mode]
```

Comment:
hard-coded link text necessary?

Reply:
Yes, the page is title "Configure Shared Storage" for both the Local and Connected mode pages. I could also just change the names on those pages to include the relevant mode though.